viewing_party/party.py

```diff
@@ -1,23 +1,151 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if title == None or genre == None or rating == None:
+        movie_data = None
+
+    else:
+        movie_data = {
+            'title': title,
+            'genre': genre,
+            'rating': rating
+            }
+
+    return movie_data
+
+
+def add_to_watched(user_data, movie):
+    user_data['watched'].append(movie)
+    return user_data
+
+
+def add_to_watchlist(user_data, movie):
+    user_data['watchlist'].append(movie)
+    return user_data
+
+
+def watch_movie(user_data, MOVIE_TITLE_1):
+    for movies in user_data.values():

```

Comment:
If we want to remove a movie from "watchlist" and move it into into "watched", we only need to loop over the "watchlist" to let us retrieve the full movie details. The outer for-loop is looping over user_data.values(), so there will be two iterations: the first one loops over "watchlist", and the second iteration loops over the "watched" list. 

The first iteration of our outer for-loop does exactly what we want: it finds the movie in the "watchlist", removes it from "watchlist", and adds the movie to the "watched" list.

The second iteration is an interesting case. We are now looping over "watched" which is already holding the movie we wanted it to. The inner loop finds the movie, removes it from the "watched" list, then adds it back to the "watched" list!

How could we structure our loops, or what keywords could we use to prevent us from iterating over the "watched" list after we've already moved the move from one list to the other?